ms-common/src/main/java/net/trajano/ms/common/Microservice.java

```diff
@@ -50,6 +53,14 @@
     public static void run(final Class<? extends Application> applicationClass,
         final String... args) {
 
+        System.setProperty("vertx.disableDnsResolver", "true");
+        System.setProperty("vertx.logger-delegate-factory-class-name", "io.vertx.core.logging.SLF4JLogDelegateFactory");
+
+        final Path path = Paths.get("logback.xml");
+        if (Files.exists(path)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this with a call to the "toFile().exists()" method [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3725)